### DISCUSSION TITLE
create-next-app install error on node 20.3.0

### DISCUSSION THREAD

**Discussion by graemen16** [vercel/next.js#51123] [Help] [answered]
### Summary

Hi, attempting to build a new app using:
` npx create-next-app nextjs-blog`
I have node 20.3.0, npm v 9.5.0 / npx v 9.5.0, running on macOS Ventura 13.4.
Output:
```
graemenelson@Gs-MBP-5 React % npx create-next-app nextjs-blog     
npm ERR! code ENOENT
npm ERR! syscall lstat
npm ERR! path /usr/local/Cellar/node/20.3.0/lib
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, lstat '/usr/local/Cellar/node/20.3.0/lib'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 
```
I confirmed there is no such directory '/usr/local/Cellar/node/20.3.0/lib'. There is a directory '/usr/local/Cellar/node/20.3.0/libexec'.
I also have node 16.22.0, and have folder '/usr/local/Cellar/Node@16/16.20.0/lib'. 

Do I need to revert to an older node version to use the create-next-app script?
Any suggestions would be most appreciated!

Update: I installed node 18 and could build the app ok, so seems related to changes in node 20.3.0, or my install of same.

**Comment by devjiwonchoi** [comment, marked as answer]
Yeah, seems like a node version issue. Would recommend you use the `LTS` version of the node, currently [18.16.0](https://nodejs.org/en).

## UPDATE

I've succeded in installing `npx create-next-app` using the `n` node manger with node v 20.3.0. Could you try to reinstall your node?

**Reply by graemen16** [reply]
Yes, I removed and reinstalled node V20.3.0 and successfully created the app. Perhaps there was indeed something amiss with my install. All is now well.
Thank you for your help on this and steering me to a happy ending!
